solr/core/src/java/org/apache/solr/search/BitsFilteredDocIdSet.java

```diff
@@ -23,15 +23,11 @@
 
 /**
  * This implementation supplies a filtered DocIdSet, that excludes all
- * docids which are not in a Bits instance. This is especially useful in
- * {@link org.apache.solr.search.Filter} to apply the {@code acceptDocs}
- * passed to {@code getDocIdSet()} before returning the final DocIdSet.
+ * docids which are not in a Bits instance.
  *
  * @see DocIdSet
- * @see org.apache.solr.search.Filter
  */
 public final class BitsFilteredDocIdSet extends FilteredDocIdSet {
```

Comment:
Since we got rid of the Filter in BitDocSet, I don't think this class is used anywhere anymore and can be deleted.

Reply:
No it isn't being used anywhere. I agree it should be deleted. I'll do that

Reply:
Nice catch Mike!